packages/opc-feedback/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@one-platform/opc-feedback",
+  "version": "0.0.1-prerelease",
+  "description": "opc-feedback component can be used to collect the feedback from the end-user",
```

Comment:
update the description - opc-feedback component follows the One Platform design system and can be used as a popup to collect feedback from the end-users.